templates/aws/aws_cloudformation_template_mysql_rds.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
The name of the file needs to change.
`mysql_rds.cfn.template.yml`

Reply:
noted